src/components/Document/DocumentTab.vue

```diff
@@ -29,8 +29,6 @@ export default {
 	@apply mr-4;
```

Comment:
Still broken, you just took the padding/margin from outside of the whole container, not from the single item.

![image](https://user-images.githubusercontent.com/27869979/98373091-0858d500-203f-11eb-8d71-59475f1c2676.png)